assets/src/edit-story/output/page.js

```diff
@@ -41,7 +41,10 @@ function OutputPage({ page, autoAdvance, defaultPageDuration }) {
     backgroundSize: '20px 20px',
     backgroundPosition: '0 0, 0 10px, 10px -10px, -10px 0px',
   };
-  const backgroundOverlayStyles = generateOverlayStyles(backgroundOverlay);
+  const backgroundOverlayStyles =
```

Comment:
I'd keep this as it used to be and do the `backgroundOverlay !== OverlayType.NONE` test later